src/main/target/CRAZYBEEF4SX1280/config.c

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Any reason why we need to accept a new legacy target? Can this new target be configured as a unified target after this PR get merged?

Reply:
IIRC flash space issues, but @phobos- knows more.

Reply:
As Hydra said, it won't fit in F411

Reply:
yup, will not fit in f411 otherwise ;) it fits into all the other unified targets with flash size greater than 512 though.